kube-web.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: caption-generator-web-deployment
```

Comment:
How about using `image-caption-generator-xxx` where xxx can be `webapp` or `web-deployment`